app/legacy/space-migration/space-migration.tsx

```diff
@@ -83,7 +83,7 @@ function Modal({onClose}) {
 
     spaces
       .migrate(({total, count, space}) => {
-        dispatch(refreshPoolNames())
+        dispatch(syncPools())
```

Comment:
Coming back to my naming comment, maybe `syncPoolNames` is my new favorite name suggestion